app/assets/javascripts/components/MentionInput.jsx

```diff
@@ -14,7 +14,7 @@ class MentionInput extends React.Component {
     }
 
     componentDidMount() {
-      Requester.get('/api/users/').then((data) => {
+      Requester.get(this.props.personType).then((data) => {
```

Comment:
this props naming is a little confusing cause I expect it to be a string like `'admin'` but it's actually a route URL